tldw_Server_API/tests/AuthNZ/test_audit_chain_integration.py

```
@@ -0,0 +1,248 @@
+    return str(tmp_path / "audit_chain_test.db")
+
+
+@pytest.fixture()
+def event_loop():
+    """Provide a fresh event loop for async tests."""
+    loop = asyncio.new_event_loop()
+    yield loop
+    loop.close()
+
+
+def _run(coro, loop=None):
```

Ruff (0.15.6)

[warning] Missing return type annotation for private function `_run`

(ANN202)